data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -191,7 +191,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "But don’t you need the gold to help rebuild Knalga? And we have plenty."
+            message= _ "But don’t you need the gold to help rebuild Knalga? We have plenty."
```

Comment:
Feels like an unnecessary change to me. My preference would be to remove everything after the question mark, however that would need rewrites elsewhere in the dialog, because the undocumented purpose of this (IIUC) is to draw the player's attention to the amount of gold that they have.